docs/docs-content/integrations/spectro-k8s-dashboard.md

```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
 <Tabs groupId="parent">
 <TabItem label="2.7.x" value="2.7.x">
 
+## Access Kubernetes Dashboard
+
+<!-- prettier-ignore-start -->
+The Kubernetes Dashboard pack automatically installs the <VersionedLink text="Spectro Proxy" url="/integrations/packs/?pack=spectro-proxy"/> pack, which serves as a reverse proxy to expose the Kubernetes dashboard and give you access to the host cluster.
+
+Connect to the cluster remotely. Refer to the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md#access-cluster-with-cli) guide for further details on how to connect using [Kubectl](https://kubernetes.io/docs/tasks/tools/). 
+
+Execute the following command from the terminal window to obtain a bearer token. Make a note of the issued token.
+
+```bash
+kubectl create token kubernetes-dashboard --namespace kubernetes-dashboard
+```
+
+The following example shows what a token value might look like.
+
+```yaml
+eyJhbGciOiJSUzI1NiIsImtpZCI6Ilg1bTg3RWM4Y1c3NnhkQ3dXbXNDUXQydVpYQklRUWoxa1BaS0ctVkVTSDQifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJrdWJlcm5ldGVzLWRhc2hib2FyZCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VjcmV0Lm5hbWUiOiJrdWJlcm5ldGVzLWRhc2hib2FyZC10b2tlbi1oNGxuZiIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJrdWJlcm5ldGVzLWRhc2hib2FyZCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjAwZTcyOWYxLTY2MzgtNGU2OC04ZGY1LWFmYTJlMmUzODA5NSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDprdWJlcm5ldGVzLWRhc2hib2FyZDprdWJlcm5ldGVzLWRhc2hib2FyZCJ9.JU4GOJNjGpkHabUyxBt_2rvtXNjpR3w238BF2oMCQUNf_ZkUGSMeAAgIKxbAuk62dtJNDaRh5yAZ9J5KthMcU6k4qVmodUOJdlvigBVNjTDEhPM-sqJus62HMtwjpvm0CX-aP_A_BqHs2yJ3OgXSX0uHmkUO1FMoZSVaRpOvx7f5bPswxd87L3npuZt4p-NJIX32-DGjBnxdANAHcWil3YHIUbDgQIdjDfN6stGU_JByvzfCJpNCWWDinr772W7iZ3uA28F8uGS0ZMd1E5e1moEFBY8BM015Qxg2Y_k7lmv9S8GMkBJyTiJNiqnwLwfsiE1ycE4Tgq_vuQfFToIMNw
+```
+
+Issue the following command to forward connections from the deployed Kubernetes Dashboard to your local port 8080.
+<!-- prettier-ignore-end -->
+
+```bash
+kubectl port-forward --namespace kubernetes-dashboard service/kubernetes-dashboard 8080:443
+```
+
+Navigate to `https://localhost:8080` in a browser of your choice to access Kubernetes Dashboard. The Dashboard login
+page appears.
+
+![Kubernetes dashboard login page](/integrations_spectro-k8s-dashboard_login-page.webp)
+
+Ensure that **Token** is selected. Then, paste the token value you issued earlier into the token field. Click **Sign
+in**. The Kubernetes Dashboard appears.
+
+## Configure Ingress
+
+Use the following steps to configure ingress for the Kubernetes Dashboard pack.
```

Comment:
Should we maybe explain that the user needs to change these parameters in the values file?

Reply:
Added this now 👍